### PR TITLE
product filters

### PR DESCRIPTION
**Pull request by rohanmathur91**
## Description:
 - Added product filters:
 - Sort by:
    -  latest 
    - low to high price
    - high to low price
 - Filter by:
    - brands
    - types of product 
 - Added product search

## Link:
https://deploy-preview-4--evolt-ecommerce.netlify.app/

## Screenshot:
image

### CODE REVIEW

File: src/components/Filters/Checkbox.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { useProduct } from "../../context";
+
+export const Checkbox = ({ id, title, value, actionType, payload }) => {
+	const { productDispatch } = useProduct();
+
+	return (
+		<label htmlFor={id} className="cursor-pointer">
+			<input
```
Comment (surbhi84) [inline src/components/Filters/Checkbox.jsx:9]: htmlFor can be removed for nested inputs.

**Review by surbhi84 — APPROVED** [review]
The code looks neat, import exports are nicely taken care of and good use of custom hooks.

File: src/components/Filters/Checkbox.jsx (hunk 1 above)
Comment (rohanmathur91) [reply src/components/Filters/Checkbox.jsx:9]: It's not nested, just a general component for checkbox, also any specific reason for why htmlFor should not be there?

**Review by aurobindoGupta — APPROVED** [review]
clean code.
sorted well-defined components.
nice styling.

File: src/components/Filters/Checkbox.jsx (hunk 1 above)
Comment (surbhi84) [reply src/components/Filters/Checkbox.jsx:9]: your label tag closes after the input, thats what i meant by nested input...in which case you dont need id attribute for label to associate with a certain input as its quite implicit.

**Pull request merged**